.github/workflows/tests.yml

```diff
@@ -0,0 +1,50 @@
+name: Tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+  tests:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, macos-latest ] # , windows-latest ]
+        ruby-version: [ "2.6", "2.7", "3.0", "3.1" ]
+
+    services:
+      cratedb:
+        image: crate/crate:nightly
+        ports:
+          - 44200:4200
+
+    name: Ruby ${{ matrix.ruby-version }} on OS ${{ matrix.os }}
+    steps:
+
+      - name: Acquire sources
+        uses: actions/checkout@v3
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+          architecture: x64
+
+      - name: Caching of Ruby gems
+        uses: actions/cache@v3
+        id: cache-gems
+        with:
+          path: vendor/bundle
+          key: gems-os=${{ matrix.os }}-ruby=${{ matrix.ruby-version }}-${{ hashFiles('**/Gemfile.lock') }}
+
+      - name: Bundle install
+        run: |
+          gem install bundler
+          bundle config path vendor/bundle
+          bundle install
+
+      - name: Run tests
+        run: bundle exec rspec
+
+      #- name: Run rubocop
+      #  run: bundle exec rubocop
```

Comment:
Rubocop isn't set up as a dependency right now, so remove it from here?

Reply:
Thanks. Fixed.